Specs/Number/Japanese/NumberModel.json

```diff
@@ -129,8 +129,8 @@
     ]
   },
   {
-    "Input": "６、５４４",
-    "Comment": "PendingValidation",
+    "Input": "6,544",
```

Comment:
Please keep the original input as well. Both encodings can happen and we should support both.